src/components/axes/categoryAxis.ts

```diff
@@ -55,11 +55,12 @@ export module Axis {
                                           offeredHeight,
                                           fakeScale,
                                           categoryScale.domain());
+
+      var desiredWidth = textResult.usedWidth + widthRequiredByTicks;
+      var desiredHeight = textResult.usedHeight + heightRequiredByTicks;
       return {
-        width : textResult.usedWidth  + widthRequiredByTicks,
-        height: textResult.usedHeight + heightRequiredByTicks,
-        wantsWidth : !textResult.textFits,
-        wantsHeight: !textResult.textFits
+        width: desiredWidth,
+        height: desiredHeight
```

Comment:
Oh, the *expressions*; that makes sense. I think each *property* should be on a different line though.